staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go

```diff
@@ -42,6 +44,8 @@ import (
 	utiltrace "k8s.io/utils/trace"
 )
 
+const StatusHeader = "X-Updated-Status"
```

Comment:
I'd probably expect this somewhere in a types.go and make sure it is documented so it can be imported by clients.